libs/image/src/KtxBundle.cpp

```diff
@@ -292,7 +292,8 @@ bool KtxBundle::getSphericalHarmonics(filament::math::float3* result) {
         return false;
     }
     float* flat = &result->x;
-    for (int i = 0; i < 9; i++) {
+    // 9 bands of 3 coefficients
+    for (int i = 0; i < 9 * 3; i++) {
```

Comment:
Could we store them as floats directly in the ktx bundle?

Reply:
Maybe, but KtxBundle is generic and can read any KTX file.  The sh metadata is a Filament-specific thing, this function was provided as a convenience only.

Reply:
I was suggesting this because it's Filament specific ^^